eZ/Publish/API/Repository/Values/Content/Location.php

```diff
@@ -56,6 +58,35 @@
     const STATUS_PUBLISHED = 1;
 
     /**
+     * Map for Location sort fields to their respective SortClauses.
+     *
+     * Those not here (class name/identifier and modified subnode) are
+     * missing/deprecated and will most likely be removed in the future.
+     */
+    const SORT_FIELD_MAP = [
```

Comment:
One thing I considered was making the method abstract, but let the constant be publicly available on the API, this would actually be consistent with current constant use.